metrics/web/test/common/presentation/navigation/metrics_page/metrics_page_test.dart

```diff
@@ -63,19 +85,24 @@ void main() {
     test(
       ".createRoute() returns a metrics page with the given route name",
```

Comment:
🏷️ 
```suggestion
      ".createRoute() returns a metrics page with the name equal to the given route name value",
```
Also, should we test the `name` getter of this class?